tst/CTA.Rules.Test/TempRules/system.linq.dynamic.json

```diff
@@ -0,0 +1,73 @@
+﻿{
```

Comment:
Does this file even get used in the test? Will this file be replaced once the real rule is merged into the porting-assistant-dotnet-datastore repo?

Reply:
We will need to remove this file once the rule is merged into the repo. This is how we get around the fact Unit tests are needed before the rule can be merged.

Reply:
I'll just leave this open as a reminder until we remove it.